Modules/Libs/QuestieHash.lua

```diff
@@ -117,7 +125,7 @@ function QuestieHash:CompareQuestHash(questId)
     else
         local oldHash = questLogHashes[questId]
         if oldHash then
-            local newHash = QuestieHash:GetQuestHash(questId)
+            local newHash = QuestieHash:GetQuestHash(questId) or oldHash
```

Comment:
This leads to wrong return values. Return may be wrong to both ways: Telling there is change when not, and no changes when there was a change.